iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java

```diff
@@ -59,20 +61,28 @@ public Map<String, List<TRegionReplicaSet>> getRegionGroupMap() {
     return regionGroupMap;
   }
 
-  public void addRegionGroup(String storageGroup, TRegionReplicaSet regionReplicaSet) {
+  public long getCreateTime() {
+    return createTime;
+  }
+
+  public void setCreateTime(long createTime) {
```

Comment:
setCreateTime only for test?

Reply:
No. This interface is also invoked at CreateRegionGroupsProcedure, where set the RegionGroup creation time.